landlab/components/lateral_erosion/tests/test_latero.py

```diff
@@ -0,0 +1,458 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed May 22 13:50:43 2019
+
+@author: abby
+
+
+Doc tests and unit tests for lateral erosion.
```

Comment:
Hooray for tests! Have you tried looking at the coverage with:

pytest --doctest-modules --cov-report term-missing --cov=.